search/search_repository.go

```diff
@@ -614,14 +615,12 @@ func (r *GormSearchRepository) listItemsFromDB(ctx context.Context, criteria cri
 	}
 
 	if parentExists != nil && !*parentExists {
-		where += ` AND
-			id not in (
-				SELECT target_id FROM work_item_links
-				WHERE link_type_id IN (
-					SELECT id FROM work_item_link_types WHERE forward_name = 'parent of'
-				)
-			)`
-
+		where += fmt.Sprintf(` AND
+			NOT EXISTS (
+				SELECT wil.target_id FROM work_item_links wil, work_item_link_types wilt
```

Comment:
Is this a cross join ? Should we use inner join here? Or are we going to rely the query plan produced by PostgreSQL here?

Reply:
@baijum yes, why wouldn't we rely on the query plan produced by PostgreSQL ? Otherwise, I'm not sure I understand your concern here. Could you elaborate, please ?

Reply:
No concern. This query and inner join produce same query plan. Please ignore.

Reply:
ok, thanks.